requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+-r requirments.txt
```

Comment:
Are you setting up a second virtual environment to run the tests?  Currently everything in $VIRTUAL_ENV/bin/python2.7/site-packages gets deployed to prod.  

I am still trying to find a good solution that keeps the unneeded code out of production